website/views.py

```diff
@@ -4515,3 +4518,42 @@ def submit_pr(request):
         return render(request, "submit_pr.html")
 
     return render(request, "submit_pr.html")
+
+
+# Global variable to store the vector store
+vector_store = None
+
+
+@api_view(["POST"])
+def question_answer_view(request):
+    question = request.data.get("question", "")
+
+    # apply validation for question , there should be a question and the question should be a string and also the length of the question should be greater than 0
+    if not question or not isinstance(question, str) or len(question) == 0:
```

Comment:
`if not question` covers that
```suggestion
    if not question or not isinstance(question, str):
```